lib/git-whistles/jira.rb

```diff
@@ -0,0 +1,54 @@
+module Git
+  module Whistles
+    module Jira
```

Comment:
I appreciate you took the time to extract common functionality! It is definitely a good idea! 

I personally don't like much module inclusion as it makes test a bit more cucumbersome by having to create extra classes or testing this from somewhere else, and also its a pain when the public methods can clash with others. To complete my comment with some constructive idea: I would rather see this guy as a class with all its methods being class methods.

What do you think about this?